src/App.js

```diff
@@ -1,13 +1,15 @@
-import './App.css';
-import { Provider } from 'react-redux';
-import store from './redux/store';
-import Test from './components/test';
+import "./App.css";
```

Comment:
Is it some kind of autoprettier? I guess it's better to disable it for test project.